dashboard/test/integration/db_query_test.rb

```diff
@@ -20,7 +20,7 @@ def setup
       level: level,
       level_source: create(:level_source, level: level)
 
-    assert_cached_queries(11) do
+    assert_cached_queries(13) do
```

Comment:
Allthethings now has bubble choice levels include in it and because of that this gets 2 more queries than before.